Dockerfiles/gadget-default.Dockerfile

```diff
@@ -97,8 +102,6 @@ COPY gadget-container/hooks/nri/conf.json /opt/hooks/nri/
 # BTF files
 COPY hack/btfs /btfs/
 
-COPY --from=bpftrace /usr/bin/bpftrace /usr/bin/bpftrace
-
```

Comment:
It is an issue. We should support bpftrace on the default image. When I added it, I decided to only add to the default image as it wasn't working on the CO-RE one because some issues with libraries. Do you know if we can make bpftrace work on the default image? Like having a statically compiled version of it.